draft-packages/avatar/KaizenDraft/Avatar/AvatarGroup.module.scss

```diff
@@ -0,0 +1,109 @@
+@import "~@kaizen/design-tokens/sass/typography";
+@import "~@kaizen/design-tokens/sass/shadow";
+@import "~@kaizen/design-tokens/sass/color";
+
+$avatar-sm: 1.25rem;
+$avatar-md: 3rem;
+$avatar-lg: 4.5rem;
+
+.AvatarGroup {
+  display: inline-flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.AvatarGroupItem,
+.AvatarCounter {
+  .small & {
+    margin-left: calc(#{$avatar-sm} / -2);
+  }
+
+  .medium & {
+    margin-left: calc(#{$avatar-md} / -2);
+  }
+
+  .large & {
+    margin-left: calc(#{$avatar-lg} / -2);
+  }
+
+  .small & [dir="rtl"] & {
+    margin-left: calc(#{$avatar-sm} / -2);
+  }
+
+  .medium & [dir="rtl"] & {
+    margin-left: calc(#{$avatar-md} / -2);
+  }
+
+  .large & [dir="rtl"] & {
+    margin-left: calc(#{$avatar-lg} / -2);
+  }
+}
+
+.AvatarGroupItem {
+  &:first-of-type {
+    margin-left: 0;
+
+    .small & {
+      [dir="rtl"] & {
+        margin-left: calc(#{$avatar-sm} / -2);
+      }
+    }
+
+    .medium & {
+      [dir="rtl"] & {
+        margin-left: calc(#{$avatar-md} / -2);
+      }
+    }
+
+    .large & {
+      [dir="rtl"] & {
+        margin-left: calc(#{$avatar-lg} / -2);
+      }
+    }
+  }
+}
+
+.AvatarCounter {
+  align-items: center;
+  background: $color-gray-300;
+  border: 3px solid $color-white;
+  border-radius: 100%;
+  box-sizing: border-box;
+  box-shadow: $shadow-small-box-shadow;
+  display: flex;
+  font-family: $typography-heading-1-font-family;
+  justify-content: center;
+  overflow: hidden;
+
+  .small & {
+    box-shadow: none;
+    border: none;
+    width: $avatar-sm;
+    height: $avatar-sm;
+    font-size: 0.5rem;
+    font-weight: $typography-heading-5-font-weight;
+    letter-spacing: $typography-heading-5-letter-spacing;
+  }
+
+  .medium & {
+    width: $avatar-md;
+    height: $avatar-md;
+    font-size: $typography-heading-5-font-size;
+    font-weight: $typography-heading-5-font-weight;
+    letter-spacing: $typography-heading-5-letter-spacing;
+  }
+
+  .large & {
+    width: $avatar-lg;
+    height: $avatar-lg;
+    font-size: $typography-heading-3-font-size;
+    font-weight: $typography-heading-1-font-weight;
+    letter-spacing: $typography-heading-3-letter-spacing;
+  }
+
+  [dir="rtl"] & {
+    margin-left: 0;
+    direction: ltr;
```

Comment:
I take it the plus is meant to stay on the left for RTL?

Reply:
I went back and forth over this one, I know with RTL layout, when a number is a negative that could be flipped onto the other side, but positive value usually just removes the symbol.

new Intl.NumberFormat("ar-EG").format(-1234);
would produce: ؜-١٬٢٣٤
but for a plus: ؜ ١٬٢٣٤

Since this isn't actually representative of a positive / negative relationship and is more a graphical element I opted to leave it as LTR.

I could potentially write up a function that runs the number through a formatter and let the output get handled that way but I think if that's the case it should be a new ticket branched off this since it may be a bit of a rabbit whole trying to get the formatting right.